azure-pipelines.vrt-pr.yml

```diff
@@ -28,6 +28,8 @@ jobs:
 
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27057&commitId=83bbed0e55681e06d9fcf15f282761c696772269&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checkbox Example Label Flex Column.chromium.png|22| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>